purpleair-aqi.js

```diff
@@ -225,27 +303,42 @@ function getAQITrend({ v1: partLive, v3: partTime }) {
   return "arrow.left.and.right";
 }
 
+/**
+ * Constructs an SFSymbol from the given symbolName
+ *
+ * @param {string} symbolName
+ * @returns {object} SFSymbol
+ */
+function createSymbol(symbolName) {
```

Comment:
I moved this function up and added docs, in order to keep the `run` function at the bottom.

Reply:
`function` declarations are hoisted, so functionally there's no need to move it. Some may prefer it for clarity, though, so 👍